bdarmory/Radar/RadarUtils.cs

```diff
@@ -0,0 +1,1562 @@
+using System.Collections.Generic;
```

Comment:
It looks like you've lower-cased the "BDA" on "BDArmory/Radar/RadarUtils.cs" and github is treating it as a new file instead of just showing the diff from the current file, which it's listing separately above.